recipes/bokeh/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set name = "bokeh" %}
+{% set version = "0.11.1" %}
+{% set checksum = "34a73d8f68e7d6faaecc9f785cac8e9f76b88a7e7aacf66307076b03b4d3c472" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Was getting the source from PyPI @bryevdv in the assumption that the Coffescript was already compiled and included for a first pass. However, it seems the JS and Coffescript for that matter are not present in the PyPI. Might be worth investigating.